app/Http/Controllers/Backend/BrouterController.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+namespace App\Http\Controllers\Backend;
+
+use App\Http\Controllers\Controller;
+use App\Models\HafasTrip;
+use App\Models\PolyLine;
+use Illuminate\Http\Client\PendingRequest;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Http;
+use Illuminate\Support\Facades\Log;
+
+abstract class BrouterController extends Controller
+{
+    public static function getHttpClient(): PendingRequest {
+        return Http::baseUrl(config('trwl.brouter_url'))
+                   ->timeout(config('trwl.brouter_timeout'));
+    }
+
+    /**
+     * @param array  $coordinates Array with Sub-Array containing the coordinates in order lat, lon (use floats!)
+     * @param string $profile
+     *
+     * @return mixed|null
+     * @throws \JsonException
+     */
+    public static function getGeoJSONForRoute(
+        array  $coordinates,
```

Comment:
Nice to have: this should be an explicit (custom) datatype, so that we're always sure what to be transferred.